mirrord/layer/src/lib.rs

```diff
@@ -455,6 +455,10 @@ fn fetch_env_vars() -> HashMap<String, String> {
         env_vars.extend(overrides.iter().map(|(k, v)| (k.clone(), v.clone())));
     }
 
+    if let Some(mapping) = setup().env_config().mapping.clone() {
+        env_vars = EnvVarsRemapper::new(mapping, env_vars).remapped();
+    }
```

Comment:
I think override should be applied after this step. Same goes for fetching env vars in the CLI

Reply:
Changed the order.